integration_tests/src/main/python/regexp_test.py

```diff
@@ -243,6 +243,20 @@ def test_re_replace():
                 'REGEXP_REPLACE(a, "TEST", NULL)'),
         conf=_regexp_conf)
 
+def test_re_replace_choice_opt():
+    gen = mk_str_gen('[a-d]{1,2}') \
```

Comment:
I don't know if this is valid.  Especially with the test pattern "ab|bc".  This is not going to ever generate "abc" which is where this pattern would/could cause issues.

Reply:
I had changed this during some debugging and forgot to change it back to the intended lengths.